R/summary_rows.R

```diff
@@ -115,6 +115,24 @@ summary_rows <- function(data,
   # the stub is necessary for summary row labels
   if (!stub_available && is.null(groups)) {
 
+    data <-
+      data %>%
+      dt_boxhead_add_var(
+        var = "rowname",
+        type = "stub",
+        column_label = list("rowname"),
+        column_align = "left",
+        column_width = list(NULL),
+        hidden_px = list(NULL),
+        add_where = "top"
+      )
+
+    # Add the `"rowname"` column into `_data`
+    data$`_data` <-
+      data$`_data` %>%
+      dplyr::mutate(rowname = rep("", nrow(data$`_data`))) %>%
+      dplyr::select(rowname, dplyr::everything())
```

Comment:
This can't possibly be right, can it? Don't you need to look in `_boxh` to see if a stub column already exists?

Reply:
This doesn't work, should it?

```
mtcars %>%
  gt(rownames_to_stub = TRUE) %>%
  summary_rows(fns = list(mean = ~mean(.)))
```

Reply:
We're good!